lib/resources/Anchors.js

```diff
@@ -19,23 +19,21 @@ export default class Anchors {
   }
 
   /**
-   * @param  {String} params.height or params.entryHash = The entry hash
-   * or height being used to queryanchors
+   * @param  {String} params.objectIdentifier = The entry hash, chain id, directory block height,
+   * directory block keymr, entry block keymr, or facotid block keymr
+   * being used to queryanchors
    * @param  {Boolean | Function} params.signatureValidation The signature validation may
    * be True/False or callback function
    * @return  {Promise} Returns a Promise that, when fulfilled, will either return an Object
    * with the Chain Info or an Error with the problem.
    */
   async get(params = {}) {
-    if (CommonUtil.isEmptyString(params.entryHash) && CommonUtil.isEmptyString(params.height)) {
-      throw new Error('entryHash or height is required.');
+    if (CommonUtil.isEmptyString(params.objectIdentifier)) {
```

Comment:
would be nice to do as Sam had suggested... add error checks for is a positive INT OR is a sha256 hash, otherwise throw an error saying that it is an invalid format or something of the sort. Use similar language to that which the other error messages provide. You can see these in the docs.